services/automated-actions.js

```diff
@@ -0,0 +1,15 @@
+import schedule from 'node-schedule'
+import firebase from 'firebase'
+import moment from 'moment'
+
+let checkDatabase = schedule.scheduleJob('* * */1 * *', () => {
+  firebase.database().ref('/games').once('value').then(function(snapshot) {
+    for (let key in snapshot.val()) {
+        var now = moment();
```

Comment:
Use `let` instead of `var`. [`no-var`](https://github.com/airbnb/javascript#references--disallow-var)